docs/catalog/cloud/Design for independent scaling of logical components.md

```diff
@@ -0,0 +1,40 @@
+---
+version: 1.0
+submitted_by: yelghali
+published_date: tbd
+category: cloud
+tags: 
+- compute
+- cloud
+- kubernetes
+- role:software-engineer
+- role:cloud-engineer
+- size:medium
+---
+
+# Scale logical components independently
+
+## Description
+A microservice architecture may reduce the compute resources required, as it allows each independent component to be scaled according to its own demand.
+
+## Solution
+
+Use cloud native projects like [Dapr](https://dapr.io) Framework to help you separate your application functionality into different microservices, to allow independent scaling of its logical components.
```

Comment:
We want to keep Pattern vendor-neutral as described [here](https://patterns.greensoftware.foundation/#principles-patterns-and-practices), so I think it's best not to mention Dapr here.